src/engraving/dom/chordrest.cpp

```diff
@@ -781,6 +781,15 @@ Breath* ChordRest::hasBreathMark() const
     return s ? toBreath(s->element(track())) : 0;
 }
 
+void ChordRest::setIsTrillCueNote(bool v)
+{
+    if (isChord()) {
+        m_isTrillCueNote = v;
+    } else {
+        m_isTrillCueNote = false;
```

Comment:
This looks too weird. It should be in the Chord class. Also, for setters, you would expect them to always change the current value, not under some conditions...